### ISSUE TITLE
Support Swift Packages

### ISSUE THREAD

**Issue by laohanme** [yagiz/Bagel#52]
Does it support SP
<img width="543" alt="image" src="https://user-images.githubusercontent.com/18119268/60710216-c13b6b00-9f44-11e9-8532-3a7045c3a25a.png">

**Comment by ricobeck** [comment]
Had the same issue and made it work:
https://github.com/ricobeck/Bagel

Had to fork Bagel and CocoaAsyncSocket to correct the location of the header files. As a bonus it now works on macOS.

Is this repo still maintained? I'd happy to provide a PR.

**Comment by yagiz** [comment]
@ricobeck Yes it's still maintained :) It would be awesome if you open a PR.

**Comment by basvankuijck** [comment]
Any update on this? 
I currently added ricobeck his repo as a dependency.
But I would rather at the original one.

**Comment by gbuela** [comment]
+1
@ricobeck can we have your PR please? 😃

**Comment by randomiser** [comment]
+1

**Comment by mohpor** [comment]
+1

**Comment by mohpor** [comment]
I have successfully migrated Bagel to SPM in #61 .
I hope it gets merged

**Comment by randomiser** [comment]
Can we get this PR merged?
